prawcore/auth.py

```diff
@@ -64,8 +64,7 @@ def authorize_url(self, duration, scopes, state, implicit=False):
             )
         if implicit and duration != "temporary":
             raise InvalidInvocation(
-                "The implicit grant flow only supports "
-                "temporary access tokens."
+                "The implicit grant flow only supports " "temporary access tokens."
```

Comment:
```suggestion
                "The implicit grant flow only supports temporary access tokens."
```

Not a big fan of changing non pre_push related changes, but if you are planning on then maybe add this fix.
Did this change come from black?

Reply:
Yes, it would appear so.